lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -0,0 +1,75 @@
+// LMS -- modules -- student notes
+// ====================
+
+// in this document:
+// --------------------
+// +notes
+// +toggling notes
+// +creating/editing notes
+// +listing notes
+
+// +notes:
+// --------------------
+// this Sass partial contains all of the styling needed for the in-line student notes UI.
+
+// +toggling notes
+// --------------------
+
+.edx-notes-visibility {
+
+  .edx-notes-visibility-error {
+    @extend %t-copy-sub2;
+    @extend %text-truncated;
+    position: relative;
+    bottom: -($baseline/20); // needed to sync up with current rogue/more complex calc utility alignment
+    max-width: ($baseline*15);
+    display: none;
+    vertical-align: bottom;
+    @include margin-right(-($baseline/4));
```

Comment:
Don't we usually have the `@includes` and `@extends` up top, first in the order?